### PR TITLE
Adding datelist

### PR DESCRIPTION
**Pull request by aclarembeau**
Adding Datelist.io, my online booking system (website in french)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1066,6 +1066,7 @@ Table of Contents
   * [Bricks Note Calculator](https://free.getbricks.app/) - a note-taking app (PWA) with a powerful built-in multiline calculator.
   * [Code Time](https://www.software.com/code-time) - an extension for time-tracking and coding metrics in VS Code, Atom, IntelliJ, Sublime Text, and more.
   * [ConfigCat](https://configcat.com) - Cross-platform feature flag service. SDKs for all major languages. Free plan up to 10 flags, 2 environments, 1 product and 5 Million requests per month. Unlimited user seats. Students get 100 flags and 100 Million requests per month for free.
+  * [datelist.io](https://datelist.io) - Online booking system for your website
```
Comment (ripienaar) [inline README.md:1069]: is there an english website? We're unable to verify anything but english sites.

Comment (ReduxGB) [reply README.md:1069]: Looked at the website. Current offers on the free plan:
Up to 5 reservations per month
1 Calendar
Email notifications
Widget for your website
Dedicated booking page

**Comment by aclarembeau** [comment]
Hello, and sorry for the delay.
Normally, the site should be in french or in english depending on your browser language. Isn't it working on your end?I

Is there anything else we should do to add our tool to this list?

Best,

**Comment by ripienaar** [comment]
OK, apparently now it has english - perhaps an addition since last reveiwed.

**Comment by ripienaar** [comment]
Please add a bit of description about what is free as seen on other listings

**Comment by aclarembeau** [comment]
No problem; it's done. 
The new description is: 
```
  * [datelist.io](https://datelist.io) - Online booking / appointment scheduling system. Free up to 5 bookings per month, includes 1 calendar
```
Is that ok on your side? 
Thanks a lot

**Comment by ripienaar** [comment]
thanks

**Pull request merged**